CONTRIBUTING.md

```diff
@@ -177,16 +177,15 @@ To keep naming patterns consistent across the project, naming patterns are enfor
 - Methods that return a variable that uses the zero value or values provided via the method MUST have the prefix `New`. For example:
   - `func NewKinesisExporter(kpl aws.KinesisProducerLibrary)` allocates a variable that uses
     the variables passed on creation.
-  - `func NewKeyValueBuilder()` may allocate internal varialbes to a safe zero value.
-- Methods that return a variable that uses non zero value(s) that impacts business logic must use `NewDefault`. For example:
+  - `func NewKeyValueBuilder()` shall allocate internal variables to a safe zero value.
```

Comment:
Should `shall` be capitalized as well?

Reply:
>MUST   This word, or the terms "REQUIRED" or "SHALL", mean that the
   definition is an absolute requirement of the specification.

Based on ^^^ from the RFC, I would expect shall to be interchangeable with must, but i dont have a strong preference for one or the other.

Reply:
long debate between must vs should. If there is no exception then we should use MUST, if there may be an edge case then we should use SHOULD